### PR TITLE
Adding a very basic openSCAP image scanning job that can be deployed …

### PR DESCRIPTION
**Pull request by etsauer**
…to clusters

@huddlesj @sabre1041

### CODE REVIEW

File: image-scanning/README.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+== Automated Deployment with Applier
+
+This capability can be deployed to any cluster using link:https://github.com/redhat-cop/openshift-applier[OpenShift Applier].
+
+[source,bash]
+----
+cd image-scanning/
+ansible-galaxy install -r requirements.yml -p galaxy
+ansible-playbook -i inventory/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
```
Comment (afcollins) [inline image-scanning/README.adoc]: Project isn't created at this point.  Does applier create a namespace?

Comment (afcollins) [reply image-scanning/README.adoc]: Also, after running command as-is, image-sign-scan-base-1-build fails:
`Error getting repository data for rhel-7-server-ose-3.7-rpms, repository not found`

Comment (etsauer) [reply image-scanning/README.adoc]: @afcollins thanks for the review! I've added the project and removed that dependency on a particular channel. Be aware however, that your minishift will require a subscription in order to build the scanner image, as it needs to be able to install `oc` from a red hat source.

File: image-scanning/templates/image-scan-base.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  description: Name Applied to the Resulting ImageStream and BuildConfig
+  value: image-sign-scan-base
+  required: true
+- description: Image stream tag for the image you'd like to use to build the application
+  name: FROM_IMAGE
+  required: true
+  value: rhel7:latest
+- description: OpenShift Major Version
+  name: VERSION
+  value: '3.7'
```
Comment (sabre1041) [inline image-scanning/templates/image-scan-base.yml]: @etsauer I tracked down the issue. Version of `image-inspector` in OCP 3.7 repos is 2.4 while version in 3.9 and in the official `image-inspector` image is 2.1. When using version 2.1, parsing errors are no longer being seen and all images run through successfully.

Can you update the version to 3.9?

**Comment by etsauer** [comment]
@sabre1041 addressed issues above.

**Comment by sabre1041** [comment]
LGTM

**Pull request merged**